src/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit/Esri.ArcGISRuntime.Toolkit.csproj

```diff
@@ -68,8 +101,16 @@
     </PackageReference>
   </ItemGroup>
   
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)Esri.ArcGISRuntime.Toolkit.Android.targets">
```

Comment:
Put in Android projitems

Reply:
Doing so doesn't work - if there, it doesn't actually get included in the NuGet package.